inf_od_nwk.py

```diff
@@ -133,24 +140,45 @@
             cv2.circle(depth_img, (int(clamp_x2), int(clamp_y2)), 5, (0, 0, 255), -1)
             cv2.imshow("clamp points", depth_img)
             key = cv2.waitKey(0)
-
+            if key & 0xFF == ord('q') or key == 27:
```

Comment:
Should probably add print statement to tell the user what to do 
```suggestion
            print ("Press q to quit or r to continue")
            if key & 0xFF == ord('q') or key == 27:
```